content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -22,15 +22,15 @@ The good practices laid out here should be read in conjunction with the general
 
 ### Least privilege
 
-Ideally minimal RBAC rights should be assigned to users and service accounts. Only permissions 
-explicitly required for their operation should be used. Whilst each cluster will be different, 
+Ideally, minimal RBAC rights should be assigned to users and service accounts. Only permissions 
+explicitly required for their operation should be used. While each cluster will be different, 
 some general rules that can be applied are :
 
  - Assign permissions at the namespace level where possible. Use RoleBindings as opposed to 
    ClusterRoleBindings to give users rights only within a specific namespace.
  - Avoid providing wildcard permissions when possible, especially to all resources.
    As Kubernetes is an extensible system, providing wildcard access gives rights
-   not just to all object types presently in the cluster, but also to all future object types
+   not just to all object types present in the cluster, but also to all future object types
```

Comment:
```suggestion
   not just to all object types that currently exist in the cluster, but also to all future object types
```

The meaning of presently here seems to be "existing in the cluster at the present time"

Reply:
True, made changes according to your suggestion.